tests/replay_rtgsfit.c

```diff
@@ -156,89 +213,38 @@ int main(int argc, char *argv[]) {
   if (t32 == NULL) {
     errorExit("test_st40pcs_mds: Error! memory t32 not allocated.");
   }
-  uint32_t sensors_idx_num;
-  if (count_lines(fn_sensors_idx, &sensors_idx_num) != 0) {
-    errorExit("Cannot count lines");
-  }
-
-  // if (N_MEAS != sensors_num) {
-  //   snprintf(error_msg, sizeof(error_msg), "Incorrect amount of sensors index "
-  //   "%d %d\n", N_MEAS, sensors_num);
-  //   errorExit(error_msg);
-  // }
-  int sensors_idx[N_MEAS];
-  if (get_int_from_file(fn_sensors_idx, N_MEAS, sensors_idx) != 0) {
-    errorExit("Cannot get array for coil sensors index.\n");
-  }
-  if (count_lines(fn_pf_coil_idx, &pf_coils_num) != 0) {
-    errorExit("Cannot count lines");
-  }
-  if (N_COIL != pf_coils_num) {
-    snprintf(error_msg, sizeof(error_msg), "Incorrect amount of pf_coils "
-      "%d %d\n", N_COIL, pf_coils_num);
-    errorExit(error_msg);
-  }
-  int32_t coil_idx[N_COIL];
-  if (get_int_from_file(fn_pf_coil_idx, N_COIL, coil_idx) != 0) {
-    errorExit("Cannot get array for coil current index.\n");
-  }
-  double meas[N_MEAS];
-  if (get_double_from_file(fn_meas, N_MEAS, meas) != 0) {
-    errorExit("Cannot get array for meas data.\n");
-  }
-  double coil_curr[N_COIL];
-  if (get_double_from_file(fn_coil_curr, N_COIL, coil_curr) != 0) {
-    errorExit("Cannot get array for coil current.\n");
+  for (i = 0; i < sensors_idx_num; ++i) {
+    if (sensors_idx[i] >= 0) {
```

Comment:
when would this condition not hold?

Reply:
This in case there is no ROG. Then it will be -1